securedrop_client/app.py

```diff
@@ -104,6 +105,9 @@ def arg_parser() -> ArgumentParser:
         type=expand_to_absolute,
         help=('SecureDrop Client home directory for storing files and state. '
               '(Default {})'.format(DEFAULT_SDC_HOME)))
+    parser.add_argument(
+        '-p', '--proxy', action='store_true',
```

Comment:
I'm somewhat not a fan of using single letters for boolean values because of the chances for confusion.

Reply:
I can remove that.